### ISSUE TITLE
change learning rate during training

### ISSUE THREAD

**Issue by xchen010** [thtrieu/darkflow#195]
In darknet, it uses "steps" policy during training, so that the learning rate can be modified **automatically** after a specified number of steps. Is it possible for darkflow to do so, and how can it be achieved? 

For instance, in darknet, if a cfg file contain following things:

learning_rate=0.0001
max_batches = 35000
policy=steps
steps=100,25000
scales=10,.1

The initial learning rate is 10e-4, and it will be changed to 10e-3 after 100 steps, and then changed back to 10e-4 after 25k steps.

**Comment by xchen010** [comment]
I modified the code a little bit to enable learning rate change during training, basically the following files:
flow
net/flow.py train
net/help.py build_train_op
But I'm new to GitHub, it seems that I made multiple pull requests ...

**Comment by thtrieu** [comment]
close due to PR https://github.com/thtrieu/darkflow/pull/216/
